asmcli/commands/install.sh

```diff
@@ -206,3 +206,68 @@ install_control_plane() {
     install_canonical_controller
   fi
 }
+
+### Installation functions ###
```

Comment:
We shouldn't need to keep these kind of comments after splitting things up into files.

Reply:
Done